README.rst

```diff
@@ -21,12 +24,38 @@ Install with pip::
 
     $ pip install flake8-coding
 
-Further options
-----------------
+Options
+-------
+
+``accept-encodings``
+~~~~~~~~~~~~~~~~~~~~
+
+A comma-separated list of acceptable source code encodings for the ``coding:``
+magic comments in files. Default is ``latin-1, utf-8``.
+
+You can pass this as a command-line argument to ``flake8``, e.g.
+``--accept-encodings=utf-8,utf-16``, or put it in your config file, e.g.:
+
+.. code-block:: ini
```

Comment:
What do you mean it causes an error? `code-block` is standard RST, you can see it renders fine on Github here: https://github.com/adamchainz/flake8-coding/tree/readme_updates

It also renders fine on PyPI.

You may need Pygments installed for it to work locally.